ankdown/ankdown.py

```diff
@@ -294,21 +308,32 @@ def cards_to_apkg(cards, output_name):
     package.write_to_file(output_name)
 
 
+def apply_arguments(arguments):
+    global CONFIG
+    if arguments.get('--configFile') is not None:
+        config_file_path = os.path.abspath(os.path.expanduser(arguments.get('--configFile')))
+        with open(config_file_path) as config_file:
+            config = json.load(config_file)
+            for attr, value in config.items():
+                CONFIG[attr] = value
+    if arguments.get('--config') is not None:
+        config = json.loads(arguments.get('--config'))
+        for attr, value in config.items():
```

Comment:
This is fine as-is but I wanted to point out that you could use one of the suggestions at https://stackoverflow.com/questions/38987/how-to-merge-two-dictionaries-in-a-single-expression

Reply:
I have improved the part, it remains 2 operations, because in this case 3 dictionaries must be merged.